### ISSUE TITLE
Digital representation of value

### ISSUE THREAD

**Issue by larryflorio** [LeXpunK-Army/infra-bill#2]
As semi-mentioned in my earlier issue, the phrase "digital representation of value" has no practical meaning in a world where the majority of money and information exists either primarily or exclusively in digital form

**Comment by lex-node** [comment]
Agreed, it's just really a bad construct that I think started in one of the first Wyoming bills and then went viral. I think what they are really trying to say is that it has to *have* value. I suggest an alternative approach [here](https://github.com/LeXpunK-Army/infra-bill/blob/main/digital-assets-major-exchange%20approach.md) (which probably isn't the greatest general definition either, but is tailored specifically for this infrastructure bill (and thus deliberately somewhat narrow)).
